src/libraries/System.IO.FileSystem/tests/File/AppendAllBytes.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO.Tests;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class File_AppendAllBytes : FileSystemTest
+    {
+
+        [Fact]
+        public void NullParameters()
+        {
+            string path = GetTestFilePath();
+
+            Assert.Throws<ArgumentNullException>(() => File.AppendAllBytes(null, new byte[0]));
+            Assert.Throws<ArgumentNullException>(() => File.AppendAllBytes(path, null));
+        }
+
+        [Fact]
+        public void InvalidParameters()
+        {
+            Assert.Throws<ArgumentException>(() => File.AppendAllBytes(string.Empty, new byte[0]));
```

Comment:
Is `new byte[0]` an invalid parameter?

Reply:
Nope, the `string.Empty` for the path parameter. We have same tests for other APIs, for example [File_ReadWriteAllBytes](https://github.com/dotnet/runtime/blob/10ce9fb494c9f2a16a757e1c8a1f902c28be4381/src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs#L25).